### PR TITLE
Fix google service that was returning nil

### PR DESCRIPTION
**Pull request by holmesm8**

### CODE REVIEW

**Comment by holmesm8** [comment]
@caachz @agallant121 @TravisBorgsmiller

File: app/poros/restaurant.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def initialize(restaurant)
   end
 
   def average_rating(google_rating)
-    return @yelp_rating if google_rating == nil
+    return @yelp_rating if google_rating == 1
```
Comment (agallant121) [inline app/poros/restaurant.rb:31]: Nice, this will handle the nil value for google ratings.

**Review by agallant121 — APPROVED** [review]

File: app/services/google_services.rb [hunk 2]
```diff
@@ -2,11 +2,19 @@ class GoogleService
...
+    @google_rating = 1
+    if phone_number == nil
+      return @google_rating
+    end
     number = phone_number[1..-1]
     api = Faraday.new("https://maps.googleapis.com/maps/api/place/findplacefromtext/json?input=%2B#{number}&inputtype=phonenumber&fields=rating&key=#{ENV["GOOGLE_API_KEY"]}")
     response = api.get
     google = JSON.parse(response.body, symbolize_names: true)
-    @google_rating = google[:candidates][0][:rating].to_s
+    if google[:status] == "INVALID_REQUEST"
```
Comment (caachz) [inline app/services/google_services.rb:13]: Great error handling!

File: app/services/google_services.rb [hunk 3]
```diff
@@ -2,11 +2,19 @@ class GoogleService
...
+      rating = 1
+    else
+      rating = google[:candidates][0][:rating].to_s
+    end
+    @google_rating = rating
```
Comment (caachz) [inline app/services/google_services.rb:18]: If we refactored we could get rid of this instance variable but its working great now

**Pull request merged**